catroid/src/androidTest/java/org/catrobat/catroid/test/scratchconverter/ScratchServerCallsTest.java

```diff
@@ -0,0 +1,230 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2017 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
```

Comment:
the copyright header in project hasnt been updated yet everywhere, so checkstyle complains that this one is not ok.
can you just change it to "2016" for this PR?